{"message":"Failed to connect to mappsdb.database.windows.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to mappsdb.database.windows.net:1433 in 15000ms\n    at ConnectionManager.connect (C:\\Repos\\Private\\mapps-io-api\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:116:17)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async ConnectionManager._connect (C:\\Repos\\Private\\mapps-io-api\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:220:24)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to mappsdb.database.windows.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2023-08-08T08:38:58.252Z"}
{"message":"Failed to connect to mappsdb.database.windows.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to mappsdb.database.windows.net:1433 in 15000ms\n    at ConnectionManager.connect (C:\\Repos\\Private\\mapps-io-api\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:116:17)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async ConnectionManager._connect (C:\\Repos\\Private\\mapps-io-api\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:220:24)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to mappsdb.database.windows.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2023-08-08T13:11:15.473Z"}
{"message":"Failed to connect to mappsdb.database.windows.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to mappsdb.database.windows.net:1433 in 15000ms\n    at ConnectionManager.connect (C:\\Repos\\Private\\mapps-io-api\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:116:17)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async ConnectionManager._connect (C:\\Repos\\Private\\mapps-io-api\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:220:24)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to mappsdb.database.windows.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2023-08-08T13:11:30.491Z"}
{"message":"Failed to connect to mappsdb.database.windows.net:1433 in 15000ms","stack":"SequelizeConnectionError: Failed to connect to mappsdb.database.windows.net:1433 in 15000ms\n    at ConnectionManager.connect (C:\\Repos\\Private\\mapps-io-api\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:116:17)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async ConnectionManager._connect (C:\\Repos\\Private\\mapps-io-api\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:220:24)","name":"SequelizeConnectionError","parent":{"message":"Failed to connect to mappsdb.database.windows.net:1433 in 15000ms","code":"ETIMEOUT"},"original":{"$ref":"$[\"parent\"]"},"level":"error","timestamp":"2023-08-08T13:11:38.528Z"}
